sdk/schemaregistry/schema-registry/src/constants.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const SDK_VERSION: string = "1.0.0-preview.1";
```

Comment:
to update this automatically, you'll want to add a `//metadata` section to package.json, e.g.

```json
"//metadata": {
    "constantPaths": [{
        "path": "src/generated/generatedClientContext.ts",
        "prefix": "packageVersion"
      },
      {
        "path": "src/constants.ts",
        "prefix": "SDK_VERSION"
      }
    ]
  },
```

Reply:
Nice.

Reply:
How does the update happen? I'd like to test it.

Reply:
Found the version tools docs.

Reply:
And done.

Reply:
@KarishmaGhiya might be able to tell you how to run it manually - there is a bot that goes through after we do a package release and bumps it

Reply:
I was able to run it manually after finding this: https://github.com/Azure/azure-sdk-for-js/blob/master/eng/tools/versioning/README.md :)

Tested changing the version and it worked great. It caught that I had no corresponding entry in CHANGELOG.md in the process.